src/protocolsupport/zplatform/impl/paper/entitytracker/PaperEntityTrackerEntry.java

```diff
@@ -0,0 +1,60 @@
+package protocolsupport.zplatform.impl.paper.entitytracker;
+
+import net.minecraft.server.v1_12_R1.Entity;
+import net.minecraft.server.v1_12_R1.EntityPlayer;
+import net.minecraft.server.v1_12_R1.EntityTracker;
+import net.minecraft.server.v1_12_R1.EntityTrackerEntry;
+import protocolsupport.api.ProtocolType;
+import protocolsupport.protocol.ConnectionImpl;
+import protocolsupport.utils.ReflectionUtils;
+import protocolsupport.zplatform.impl.spigot.entitytracker.SpigotEntityTrackerEntry;
+
+import java.lang.reflect.Field;
+import java.util.Map;
+
+public class PaperEntityTrackerEntry extends SpigotEntityTrackerEntry {
+	private static Field TRACKED_PLAYER_MAP_FIELD;
+	private static Field TRACKER_ENTITY_FIELD;
+	private Map<EntityPlayer, Boolean> trackedPlayerMap;
+
+	static {
+		TRACKED_PLAYER_MAP_FIELD = ReflectionUtils.getField(EntityTrackerEntry.class, "trackedPlayerMap");
+		TRACKER_ENTITY_FIELD = ReflectionUtils.getField(Entity.class, "tracker");
+	}
+
+	public PaperEntityTrackerEntry(Entity entity, int trackRange, int viewDistance, int updateInterval, boolean updateVelocity) {
+		super(entity, trackRange, viewDistance, updateInterval, updateVelocity);
+		setEntityTracker(entity, this); // Paper
+		try {
+			trackedPlayerMap = (Map<EntityPlayer, Boolean>) TRACKED_PLAYER_MAP_FIELD.get(this);
+		} catch (IllegalAccessException e) {
+			e.printStackTrace();
+		}
+	}
+
+	@Override
+	protected void addTrackedPlayer(EntityPlayer player) {
+		this.trackedPlayerMap.put(player, true);
```

Comment:
You are adding to the trackedPlayerMap for sure, and adding to the tracked PE and Default players will make some packets to be send. BUT... don't methods such as `void clear(final EntityPlayer entityplayer)` or even the normal 'public void broadcast(final Packet<?> packet)' depend on the trackedPlayers set? As far as I can see, this set will always be empty.

Reply:
hmmmm yeah, I think I need to test it then.

I just need to figure out an way to see if it would really break or not.

Reply:
@MrPowerGamerBR have you figured it out yet? If Shevchik is okay with this, I am too :P

Reply:
@7kasper yeah, don't you remember the pig sacrifice I did? rip smol piggy

You asked me to test if the entity metadata was still working after this change, and it was working, I tried igniting a pig and it did work.

Reply:
Yes I do remember that.